### DISCUSSION TITLE
Nested accounts usage clarification

### DISCUSSION THREAD

**Discussion by Virgus** [floccusaddon/floccus#1206] [General]
Dear all, I just started experimenting with floccus, both via Google Drive and WebDav storage and I foresee the need for multiple accounts as each browser has a "Bookmark toolbar" and a "Bookmark menu" and my Android device is an extra source of urls that I need to access from PC.

To keep things clean I'd like to keep the three separate and I'm wondering how to set up a configuration that allows me to:
- share "Bookmark menu" contents amongst every browser/device
- upload to webdav each "Bookmark toolbar" of each browser and eventually "pull down" the toolbar xbel file when needed on another browser.
- share the "android.xbel" between the smartphone and some browsers (in their "bookmarks menu" I suppose) but keeping them separate from the rest.

I'm wondering which is the easiest way to manage all this, and I need a clarification (usage examples would help) of "Nested accounts" configuration which might be necessary for this kind of setup.

Thanks for this tool and for any hint/help you might give me.
V.

**Comment by marcelklehr** [comment]
Ahoy @Virgus 

> share "Bookmark menu" contents amongst every browser/device

For this you can create an account on each browser that all link from Bookmark Menu to one webdav file. If one browser doesn't have Bookmarks Menu you can create a new folder for that and link that.

> upload to webdav each "Bookmark toolbar" of each browser and eventually "pull down" the toolbar xbel file when needed on another browser.

So, you don't want to merge the toolbars? This is not very convenient yet with webdav. You could set up separate xbel files for each browser and link all browsers to all of them, but it would get confusing quickly, I suppose.

> share the "android.xbel" between the smartphone and some browsers (in their "bookmarks menu" I suppose) but keeping them separate from the rest.

You can create a new folder in the bookmarks menu and link that to android.xbel with a new account. Make sure to disable nested accounts for that account, so the android bookmarks will not end up in your bookmarks menu xbel file.

I hope this helps.

**Reply by Virgus** [reply]
Thanks, yes it helps indeed. Could you please give me a typical use of nested folders feature ? I'm still not getting how it works exactly...

**Reply by marcelklehr** [reply]
Say you have a work folder containing all bookmarks that you need for work. On your work computer you only sync this folder. On your private computer you also sync this folder, but you also sync the entire bookmarks toolbar. If you want your work bookmarks to show up on your phone, you can now recreate the two sync accounts you setup on your private computer, or you can allow nested sync in order to sync the work folder along with the bookmarks toolbar folder to your phone. It depends a bit on your taste, if you'd rather have synced folders that are nested isolated or not.

**Reply by Virgus** [reply]
Thanks, I'll try to follow your example to make a test setup and play a bit with these options. Many thanks again ! Cheers, V.
